monitoring/README.md

```diff
@@ -1,112 +1,87 @@
-# Monitoring Tools for KubeStellar (Prometheus, Grafana and Node Exporter)
+# Monitoring Tools for KubeStellar Control Plane
 
-<img src="images/kubestellar-architecture-aws.png" width="200%" height="200%" title="kubestellar-architecture-aws">
+<img src="images/ks-monitoring-setup.png" width="80%" height="80%" title="kubestellar-monitoring-setup">
 
 
 ### Description
-This example shows how to deploy monitoring tools (Prometheus, Grafana and node exporter) for KubeStellar components (core and edge regions) - see architecture image above. Prometheus server is deployed in the core region running the KCP server alongside the components for KCP edge. A Prometheus agent is deployed in the edge regions running the edge workload execution cluster (wec)
+This example shows how to deploy monitoring tools (Prometheus, Grafana and Pyroscope) for KubeStellar control plane components - see architecture image above. These instructions can be used to deploy monitoring tools to any KubeStellar system. If you already have the monitoring tools installed in your environment, you can skip to `step-2`.
 
-1. Create your hosts file with the list of target hosts (KCP server & Edge Workload Execution Clusters (WEC))
+1. Install the monitoring tools:
 
-```
-[kcp-server]
-192.168.56.2
+Starting from a local directory containing the git repo, do the following.
 
-[edge-wecs]
-192.160.56.10
-192.160.56.12
+```bash
+$ cd monitoring
+$ ./install-ks-monitoring.sh
 ```
 
-2. Configure the Prometheus targets endpoints:
+Optionally, check the deployment of the monitoring tools:
+
+```bash
+$kubectl -n ks-monitoring get pods
```

Comment:
Missing space here between "$" and "kubectl".

Reply:
Fixed!